test/SILOptimizer/access_enforcement_noescape.swift

```diff
@@ -372,15 +339,16 @@ func readBoxWriteInout() {
 // CHECK-LABEL: } // end sil function '_T027access_enforcement_noescape17readBoxWriteInoutyyFyycfU_'
 
 // Error: inout cannot be captured.
-// func inoutReadBoxWriteInout(x: inout Int) {
-//   let c = { _ = x }
-//   doOneInout(c, &x)
-// }
+// Verification disabled because is suppresses other errors.
+//func inoutReadBoxWriteInout(x: inout Int) {
+//  disabled-error@+1{{escaping closures can only capture inout parameters explicitly by value}}
+//  let c = { _ = x }
```

Comment:
Right. Those commented out tests aren't really specific to this PR. I just happened to notice  that they broke '-verify' if I try to enable them (not sure why). I'll try moving them to a different file and getting them enabled--follow up PR later today.